lib/mocha.js

```diff
@@ -373,6 +377,20 @@ Mocha.prototype.timeout = function(timeout) {
 };
 
 /**
+ * Set the number of times to retry failed tests.
+ *
+ * @param {Number} retry times
+ * @return {Mocha}
```

Comment:
ah, I fixed that, but turns out there're actually a bunch of other methods w/ dupe annotations (`timeout`, `enableTimeout`...)